randovania/data/json_data/prime2.txt

```diff
@@ -4072,7 +4080,7 @@ Asset id: 2146386747
   * Light Door to Battleground/Door to Double Path (Bottom)
   > Safe Zone (Bottom)
       All of the following:
-          Activate Safe Zone
+          Morph Ball and Morph Ball Bomb and Space Jump Boots and Bomb Space Jump (Intermediate) and Activate Safe Zone
```

Comment:
What does this line mean? By this, you're forced to have bombs and sj to cross this path. This is not correct.